wrapanapi/systems/rhevm.py

```diff
@@ -1079,3 +1079,18 @@ def import_template(self, edomain, sdomain, cluster, temp_template):
             cluster=types.Cluster(id=cluster_id),
             template=types.Template(id=export_template.id)
         )
+
+    @property
+    def _vnic_profile_service(self):
```

Comment:
I see it is used in one place. do you think such property is necessary ?
I'd turn it into public property If so

Reply:
This is also trying to remain consistent with the existing code e.g. `_storage_domains_service`, `_templates_service` etc. I agree that it probably isn't necessary, but future methods may make use of it.

Reply:
yup, but there is a chance that there will be no such future methods.
so, I prone to not creating such "future" methods  :)